security/ssh_certificate.ksy

```diff
@@ -143,19 +142,20 @@ types:
       NOT normally be stored in the string. For example: the US-ASCII
       string "testing" is represented as 00 00 00 07 t e s t i n g. The
       UTF-8 mapping does not alter the encoding of US-ASCII characters.
-    -webide-representation: '{len_value:dec} bytes'
+    doc-ref: 'https://tools.ietf.org/html/rfc4251#section-5'
     seq:
       - id: len_value
         type: u4
       - id: value
         size: len_value
+    -webide-representation: '{len_value:dec} bytes'
```

Comment:
I don't know what ksylint said, but I personally think that it makes sense to put the `-webide-representation` at the top of the type definition, i.e. below `doc` and `doc-ref`, but above `seq` and `instances`. On top of that, it usually makes sense to stick to the [order in the style guide](https://doc.kaitai.io/ksy_style_guide.html#type):

> ## 2. Order of sections in a type spec
> 
> Use the following order of sections:
> 
> - `meta`, if present, MUST go first
> 
> - `doc`
> 
> - `doc-ref`
> 
> - `seq`
> 
> - `instances`, `types`, `enums` — use one’s best judgement to order these 3 to maximize readability